### ISSUE TITLE
`Error: Element type is invalid` with react-native-maps v1.0.0

### ISSUE THREAD

**Issue by ptreitler** [bramus/react-native-maps-directions#195]
After updating to v1.0.0 of react-native maps I got the following error:

```
 ERROR  Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `MapViewDirections`.
```

**Comment by nicolasdevienne** [comment]
@bramus the fix seems not to be included in the last version 1.1.0 ... When I install the package thanks to `yarn add react-native-maps-directions`, the fix is missing from file src/MapViewDirections.js ...

**Comment by Judefabi** [comment]
same here @nicolasdevienne , however, if you navigate to /node_modules/react-native-maps-directions/src/MapViewDirections.js and edit mapviewdirections.js with the code above it work. when you build however it restores the default code and it will fail again

**Comment by nicolasdevienne** [comment]
hi @Judefabi ! A fix has been merged : https://github.com/bramus/react-native-maps-directions/pull/196 but does not appear when we install the package...

**Comment by bramus** [comment]
I’ll push a new release later today.

**Comment by bramus** [comment]
Released as 1.9.0

**Comment by rellaElla** [comment]
Seeing this issue in version 1.9.0 using react-native version 0.69.4
